src/app/games/tafl/tafl.component.ts

```diff
@@ -117,7 +117,7 @@ export abstract class TaflComponent<R extends TaflRules<M, S>, M extends TaflMov
     private pieceBelongToCurrentPlayer(coord: Coord): boolean {
         const state: S = this.getState();
         const player: Player = state.getCurrentPlayer();
-        return state.getRelativeOwner(player, coord) === RelativePlayer.PLAYER;
+        return this.role === player && state.getRelativeOwner(player, coord) === RelativePlayer.PLAYER;
```

Comment:
non ! ça interdit de jouer le joueur deux du coup :p !
Si tu joue humain contre humain, this.role reste Player.ZERO !

Reply:
Donc:
  - J'ai ajouté au ticket rename: this.role -> this.pointOfView
  - Je propose d'ajouter un ticket [W] [easy] GameWrapper: add this.isInteractive boolean to know whether to highlight pieces, qu'en dis tu ? Et je kill d'ici les changements tafl + epaminondas

Reply:
j'ai fait les deux, tu peux cocher dans le ticket rename ou me dire de le faire si tu es satisfait !